### PR TITLE
4 get articles

### PR DESCRIPTION
**Pull request by rushRobBlob**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -86,4 +71,26 @@ describe('GET /api/articles/:article_id', () => {
...
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(String)
+            }
+            expect(body.articles).toHaveLength(13);
+            body.articles.forEach((article) => {
+                expect(article).toMatchObject(testArticle);
+            })
+        })
```
Comment (itskatherine) [inline __tests__/app.test.js:95]: Solid test 🧪 , great that you've checked the length of the returned array so you're not risking a false positive if there's an empty array 👍 
Could do with also checking that there is no body property on the returned articles objects

File: models/app.models.js [hunk 2]
```diff
@@ -14,4 +14,17 @@ exports.retrieveArticlesById = (articleId) => {
...
+}
+
+exports.retrieveAllArticles = () => {
+    const query = `
+    SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comment_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON comments.article_id = articles.article_id
+    GROUP BY articles.article_id
+    ORDER BY articles.created_at DESC;
+    `;
```
Comment (itskatherine) [inline models/app.models.js:26]: Nice use of SQL 🦭

**Review by itskatherine — COMMENTED** [review]
Hi Robbie, looks good have a look over my comments then you're good to merge 👍

File: __tests__/app.test.js (hunk 1 above)
Comment (rushRobBlob) [reply __tests__/app.test.js:95]: Aweesome thanks, will add that body test before i merge it. Cheers

**Pull request merged**